util/time.go

```diff
@@ -8,19 +8,19 @@ import (
 )
 
 const (
-	// A tag used to set custom AWS Tags to resources that do not support `created at` timestamp> - EIP & ECS Clusters.
+	// FirstSeenTagKey A tag used to set custom AWS Tags to resources that do not support `created at` timestamp> - EIP & ECS Clusters.
```

Comment:
Are we using this value outside of this module? Should it be lower camel case still?

Reply:
Yeah this is used outside of this module.